### PR TITLE
fetch data and add event for read comic

### PR DESCRIPTION
**Pull request by anhmv-1205**
#### Screenshots (nếu có)
S90416-224051

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/java/com/sun_asterisk/comics_01/screen/read/adapter/ContentAdapter.java [hunk 1]
```diff
@@ -19,6 +19,11 @@ public ContentAdapter() {
     }
 
     public void setData(List<String> links) {
+        if (links != null) {
+            mLinks.clear();
+            mLinks = links;
```
Comment (thaohq-0814) [inline app/src/main/java/com/sun_asterisk/comics_01/screen/read/adapter/ContentAdapter.java]:
```suggestion
            mLinks.addAll(links);
```

**Review by thaohq-0814 — APPROVED** [review]

File: app/src/main/java/com/sun_asterisk/comics_01/screen/comic/ComicDetailActivity.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ private void initView() {
 
     private void initData() {
         if (mComic != null) {
-            Glide.with(this).load(mComic.getThumbnail()).centerCrop().into(mImgThumbnail);
+            Glide.with(this).load(mComic.getThumbnail()).centerCrop().placeholder(R.drawable.img_not_found).into(mImgThumbnail);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/comics_01/screen/comic/ComicDetailActivity.java]: reformat code

**Comment by anhmv-1205** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**